easy-ui-react/src/Tooltip/Tooltip.tsx

```diff
@@ -124,7 +124,10 @@ export function Tooltip(props: TooltipProps) {
 
   return (
     <>
-      {React.cloneElement(children, { ...triggerProps, ref: triggerRef })}
+      {React.cloneElement(children, {
+        ...mergeProps(triggerProps, children.props),
```

Comment:
`useTooltipTrigger()` from React Aria sets some `triggerProps` to `undefined`, like `tabIndex`, which overrides values provided by the consumer. this ensures props provided by the consumer are merged into the trigger props provided

Reply:
**question**: do you know if its coming from their `useFocusable` hook or somewhere else?

Reply:
yes i believe it's coming from `useFocusable`. i looked at the code and the intention is to simply "reset" `tabIndex` by assigning it to `undefined` but because that comes back as a key on the returned props it doesn't allow for proper merging with `cloneElement()`. another little quirk with react aria